packages/blade/src/storybook-recipes/AccessibilityInterop/AccessibilityInteropDemo.web.tsx

```diff
@@ -39,6 +41,7 @@ const Checkbox: React.FC<{ label: string; checked?: boolean; disabled?: boolean
         {...checkboxAccessibilityProps}
         tabIndex={0}
         style={{
+          // @ts-ignore
```

Comment:
Why is this needed? And why did this uncover now? Due to TS upgrade?

Reply:
This is Due to TS upgrade. 

It has to do with Box's style prop definition different from react-native to web. So ts-ignored for now.